README.md

```diff
@@ -7,9 +7,13 @@ products:
 - azure-marketplace
 ---
 
-# Azure Marketplace Kubernetes App samples
+# Azure Marketplace Kubernetes App quick starting Guide
+
+## Getting started guide
+
+- [Getting Started: Publishing a Kubernetes App on Azure Marketplace](getting-started/GettingStarted.md)
+- [Microsoft Defender: Vulnerability Pre-Scan Check](getting-started/Vulnerability-Scan.md)
 
-This repository contains samples for creating Kubernetes Applications in Azure Marketplace.
```

Comment:
Can you add text "This repository contains getting started guide and samples for creating Kubernetes Applications in Azure Marketplace."